region/maridia/inner-yellow/Bug Sand Hole.json

```diff
@@ -819,10 +874,44 @@
       "flashSuitChecked": true,
       "devNote": "Artificial morph will allow Samus not to fall; real Morph is not sufficient."
     },
+    {
+      "link": [3, 4],
+      "name": "Jump Through Room",
+      "entranceCondition": {
+        "comeInJumping": {
+          "speedBooster": "any",
+          "minTiles": 1
+        }
+      },
+      "requires": [
+        "canPrepareForNextRoom",
+        "canTrickyJump",
+        "canPlayInSand"
+      ],
+      "note": [
+        "Jump through the door to cross most of the room, then jump off of the sand to cross the rest.",
+        "Crossing the room with enough speed to cross with one jump will set Samus up to be grabbed by the Yapping Maw."
```

Comment:
```suggestion
        "With enough speed, crossing the whole room without landing in the sand is also possible, but Samus must still jump quickly after landing to avoid being grabbed by the Yapping Maw."
```